notebooks/01b-scarlet-measure.ipynb

```diff
@@ -397,9 +397,9 @@
  "metadata": {
   "celltoolbar": "Aucun(e)",
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "btk38",
    "language": "python",
-   "name": "python3"
+   "name": "btk38"
```

Comment:
We should avoid changing the metadata of the notebooks ; what did we want to do about that @ismael-mendoza ?

Reply:
oh yeah... I wonder why this hasn't happend before. Hmm let me see...

Reply:
Oh I think that's because I used conda kernel when running this notebook. So, I switched default "python3" to "btk38"

I will fix it!

Reply:
thanks!